File: app/src/main/java/mufilbito/com/videoplayer/retrohit/JsonPlaceHolderApi.java
package mufilbito.com.videoplayer.retrohit;


import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface JsonPlaceHolderApi {

    @GET("posts")
    Call<List<Posts>> getposts();

    @GET("posts/{id}/comments")
    Call<List<Comments>> getcomments(@Path("id") int postid);

}

File: app/src/main/java/mufilbito/com/videoplayer/kotlinClasses/retrohit/JsonPlaceHolderApi.kt
package mufilbito.com.videoplayer.kotlinClasses.retrohit

import mufilbito.com.videoplayer.retrohit.Comments
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface JsonPlaceHolderApi {
    @GET("posts")
    abstract fun getposts(): Call<List<Model.Posts>>

    @GET("posts/{id}/comments")
    abstract fun getcomments(@Path("id") postid: Int): Call<List<Model.Comments>>

}